client/checkout/utils/upe.js

```diff
@@ -119,6 +119,18 @@ export const getUpeSettings = () => {
 		};
 	}
 
+	if ( window.wcpayCustomerData ) {
```

Comment:
It took me a bit to find the other area where we check for `window.wcpayCustomerData` and pass such data to Stripe, and to identify if there are any opportunities for improvement - but I think this is the only way, good find!